docs/deploying a sample wildfly application.md

```diff
@@ -0,0 +1,144 @@
+# Deploying a sample WildFly application
```

Comment:
cool thanks. Remember, this file will be used to generate HTML. With the spaces in it, the framework might need to escape spaces (%20) which makes ugly URLs. Also, if we ever want to process some of these files with a script, we might run into trouble as well.